static/js/lib/ckeditor/plugins/ResourceEmbed.ts

```diff
@@ -185,22 +220,12 @@ class ResourceEmbedEditing extends Plugin {
  * into the editor. These are rendered to Markdown as `{{< resource UUID >}}`
  * shortcodes.
  */
-export default class ResourceEmbed extends Plugin {
+export default class ResourceEmbed extends CKEPlugin {
   static get pluginName(): string {
     return "ResourceEmbed"
   }
 
-  static get requires(): Plugin[] {
```

Comment:
using `extends Plugin` was confusing Typescript. TS thought this was referring to the global, deprecated [Plugin](https://developer.mozilla.org/en-US/docs/Web/API/Plugin) interface.